app/cortexlib/ipblockcheck.py

```diff
@@ -60,3 +53,13 @@ def validate_config(self):
             if not isinstance(v, str):
                 msg = "%s is not the correct type!  Should be string!" % str(v)
                 raise ValueError(msg)
+
```

Comment:
These two functions, string_list and string_bool are also in the other test. Should make into a lib so there's no duplicate code